src/core/thread/mle_types.hpp

```diff
@@ -263,6 +263,18 @@ enum
 
 #endif
 
+#if OPENTHREAD_CONFIG_MLR_ENABLE
+
+/**
+ * MLR constants
+ *
+ */
+enum
+{
+    kMlrTimeoutMin = 300, //< In seconds.
```

Comment:
does this constant require `OPENTHREAD_CONFIG_MLR_ENABLE`?

Reply:
It's in the scope `OPENTHREAD_CONFIG_MLR_ENABLE` now, but it's not a must. I have defined `kMlrTimeoutMin` in the BBR validation PR. Will adjust when resolve conflicts with that PR.

Reply:
I changed the configuration for these constants to be `OPENTHREAD_CONFIG_BACKBONE_ROUTER_ENABLE || OPENTHREAD_CONFIG_MLR_ENABLE` because some are required by BBR and some are required by MLR and DUA features. 

I think it's fine to expose these constants to a even larger scope (e.g. OPENTHREAD_CONFIG_THREAD_VERSION >= OT_THREAD_VERSION_1_2).